addons/iap/tools/iap_tools.py

```diff
@@ -138,6 +109,9 @@ def iap_jsonrpc(url, method='call', params=None, timeout=15):
     Calls the provided JSON-RPC endpoint, unwraps the result and
     returns JSON-RPC errors as exceptions.
     """
+    if hasattr(threading.current_thread(), 'testing') and threading.current_thread().testing:
```

Comment:
@lobaudoux Why has this been done, because it is weird we need to re-patch this if in external tests (which are now active for l10n in the nightly runbot) we want to test the iap.  (and it seems you are patching below as well)